aws-shared-1/main.tf

```diff
@@ -97,6 +97,28 @@ module "aws_az_1e" {
   workers_org_subnet_cidr = "${var.workers_org_subnet_1e_cidr}"
 }
 
+resource "aws_route53_record" "workers_org_nat" {
+  zone_id = "${var.travisci_net_external_zone_id}"
+  name = "workers-nat-org-${var.env}-${var.index}.aws-us-east-1.travisci.net"
+  type = "A"
+  ttl = "300"
+  records = [
```

Comment:
If we have >1 AZ defined but each AZ's infrastructure is discrete, how can we be sure that we get any particular AZ at any time? or, alternatively, do we not care?

would we want to have a separate DNS entry for each AZ as well as one overarching DNS entry for if you wanted to hit Any Ol' AZ (which would theoretically be this record)?